codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/EndpointTestGenerator.kt

```diff
@@ -34,33 +35,26 @@ class EndpointTestGenerator(
     private val ctx: ProtocolGenerator.GenerationContext
 ) {
     fun render(writer: SwiftWriter): Int {
-        if (endpointTest.testCases.isEmpty()) {
-            return 0
-        }
+        if (endpointTest.testCases.isEmpty()) { return 0 }
 
         writer.addImport(ctx.settings.moduleName, isTestable = true)
-        writer.addImport(SwiftDependency.CLIENT_RUNTIME.target)
-        writer.addImport(AWSSwiftDependency.AWS_CLIENT_RUNTIME.target)
-        writer.addImport(AWSSwiftDependency.AWS_COMMON_RUNTIME.target)
         writer.addImport(SwiftDependency.XCTest.target)
-        writer.addImport(SwiftDependency.SMITHY_TEST_UTIL.target)
-        writer.addImport(SwiftDependency.SMITHY_HTTP_API.target)
 
         // used to filter out test params that are not valid
         val endpointParamsMembers = endpointRuleSet?.parameters?.toList()?.map { it.name.name.value }?.toSet() ?: emptySet()
 
         var count = 0
-        writer.openBlock("class EndpointResolverTest: \$L {", "}", XCTestTypes.XCTestCase) {
+        writer.openBlock("class EndpointResolverTest: \$N {", "}", XCTestTypes.XCTestCase) {
```

Comment:
Symbols are to be written with the `$N`, `$T`, or `$D` format specifiers; these are specialized for writing symbols.  `$L` writes the symbol as a string literal and does not track the symbol's imports or dependencies.